rpd_generator/bdl_structure/bdl_commands/construction.py

```diff
@@ -57,6 +60,28 @@ def populate_data_elements(self):
         # This u_factor will be adjusted when used for a surface based on Ext/Int/Underground Wall air film resistances
         self.u_factor = self.try_float(self.get_inp(BDL_ConstructionKeywords.U_VALUE))
 
+        # Determine if the constructions is assigned to multiple slabs on different Z coordinates
+        z_coordinate_set = set()
+        for underground_wall_name in self.rmd.undg_wall_names:
+            underground_wall = self.get_obj(underground_wall_name)
+            if (
+                underground_wall.get_inp(BDL_UndergroundWallKeywords.CONSTRUCTION)
+                != self.u_name
+            ):
+                continue
+
+            tilt = underground_wall.try_float(
+                underground_wall.get_inp(BDL_UndergroundWallKeywords.TILT)
+            )
+            if tilt is None or tilt <= 120:
```

Comment:
I deleted my comment but SOG has tilt of 180 as far as I can see so I am not understanding this. Maybe I don't understadn the